### PR TITLE
ENG-970: Update rel.cd/pt.cd/pg.cd Jenkins to 2.263.3 and their OS

### PR DESCRIPTION
**Pull request by Sudokamikaze**
Deployed across all mentioned

Used latest AMI

### CODE REVIEW

File: IaC/rel.cd/JenkinsStack.yml [hunk 1]
```diff
@@ -712,28 +706,33 @@ Resources:
...
+                    rm -rf /etc/letsencrypt
+                    rsync -aHSv /mnt/ssl_backup/ /etc/letsencrypt/
+                    certbot renew
+                  else
+                    certbot --debug --non-interactive certonly --agree-tos --register-unsafely-without-email --webroot -w /usr/share/nginx/html --keep -d $JENKINS_HOST
+                  fi
                   ln -f -s /etc/letsencrypt/live/$JENKINS_HOST/fullchain.pem /etc/nginx/ssl/certificate.crt
                   ln -f -s /etc/letsencrypt/live/$JENKINS_HOST/privkey.pem   /etc/nginx/ssl/certificate.key
-                  printf '#!/bin/sh\ncertbot renew\nservice openresty reload\n' > /etc/cron.daily/certbot
+                  printf '#!/bin/sh\ncertbot renew\nsystemctl reload openresty\nrm -rf  /mnt/ssl_backup\nrsync -aHSv /etc/letsencrypt/ /mnt/ssl_backup/\n' > /etc/cron.daily/certbot
```
Comment (delgod) [inline IaC/rel.cd/JenkinsStack.yml:723]: I believe `systemctl restart openresty` should be used here
```suggestion
                  printf '#!/bin/sh\ncertbot renew\nsystemctl restart openresty\nrsync -aHSv --delete /etc/letsencrypt/ /mnt/ssl_backup/\n' > /etc/cron.daily/certbot
```

File: IaC/rel.cd/JenkinsStack.yml [hunk 2]
```diff
@@ -712,28 +706,33 @@ Resources:
...
-                  service openresty start
+                  systemctl enable --now openresty
               }
 
               setup_letsencrypt() {
                   install -d /usr/share/nginx/html
-                  certbot --debug --non-interactive certonly --agree-tos --register-unsafely-without-email --webroot -w /usr/share/nginx/html --keep -d $JENKINS_HOST
+                  if [[ -d /mnt/ssl_backup ]]; then
+                    rm -rf /etc/letsencrypt
+                    rsync -aHSv /mnt/ssl_backup/ /etc/letsencrypt/
```
Comment (delgod) [inline IaC/rel.cd/JenkinsStack.yml:716]: JFYI `rm` not needed if `--delete` specified 
```suggestion
                    rsync -aHSv --delete /mnt/ssl_backup/ /etc/letsencrypt/
```

File: IaC/rel.cd/JenkinsStack.yml [hunk 3]
```diff
@@ -603,13 +597,13 @@ Resources:
                   sed -i 's^"-Djava.awt.headless=true"^"-Djava.awt.headless=true -Xms2048m -Xmx4096m -server -XX:+AlwaysPreTouch -verbose:gc -Xloggc:$JENKINS_HOME/gc-%t.log -XX:NumberOfGCLogFiles=5 -XX:+UseGCLogFileRotation -XX:GCLogFileSize=20m -XX:+PrintGC -XX:+PrintGCDateStamps -XX:+PrintGCDetails -XX:+PrintHeapAtGC -XX:+PrintGCCause -XX:+PrintTenuringDistribution -XX:+PrintReferenceGC -XX:+PrintAdaptiveSizePolicy -XX:+UseConcMarkSweepGC -XX:+ExplicitGCInvokesConcurrentAndUnloadsClasses -XX:+CMSParallelRemarkEnabled -XX:+ParallelRefProcEnabled -XX:+CMSClassUnloadingEnabled -XX:+ScavengeBeforeFullGC -XX:+CMSScavengeBeforeRemark -XX:NewSize=512m -XX:MaxNewSize=3g -XX:NewRatio=2 -Dorg.jenkinsci.plugins.durabletask.BourneShellScript.HEARTBEAT_CHECK_INTERVAL=600"^' /etc/sysconfig/jenkins
                   echo JENKINS_HOME=/mnt/$JENKINS_HOST \
                       | tee -a /etc/sysconfig/jenkins
-                  chkconfig jenkins on
-                  service jenkins start
+                  systemctl enable jenkins
+                  systemctl start jenkins
```
Comment (delgod) [inline IaC/rel.cd/JenkinsStack.yml:601]:
```suggestion
                  systemctl enable --now jenkins
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: IaC/rel.cd/JenkinsStack.yml (hunk 3 above)
Comment (Sudokamikaze) [reply IaC/rel.cd/JenkinsStack.yml:601]: It redirects to `chkconfig`, enables the service, but doesn't start

File: IaC/rel.cd/JenkinsStack.yml (hunk 2 above)
Comment (Sudokamikaze) [reply IaC/rel.cd/JenkinsStack.yml]: Fixed

File: IaC/rel.cd/JenkinsStack.yml (hunk 1 above)
Comment (Sudokamikaze) [reply IaC/rel.cd/JenkinsStack.yml]: Fixed

**Review by delgod — APPROVED** [review]

**Pull request merged**